### DISCUSSION TITLE
Smaller projects you know have good first issues / need contributors?

### DISCUSSION THREAD

**Discussion by Bestra** [community/events#63] [Community]
I'm listening to the "Spotlight on Maintainers" community track, and they're talking about how a newer open source project is great because things aren't so complicated as to make contributing prohibitive yet. There's still plenty of low hanging fruit for new people to come in and work on.

Does anybody here have projects they know fit this description, or projects that are looking for contributors? Or maybe projects you've really enjoyed contributing to in the past?

**Comment by ghostinhershell** [comment]
Hey @Bestra! 👋 

Here are a couple of projects that are tailored towards beginners looking to make their first open-source contribution:

- [How to choose (and contribute to) your first open source project](https://github.com/collections/choosing-projects)
- The [First Contributions](https://github.com/firstcontributions/first-contributions) repo
- FreeCodeCamp's [How to Contribute to Open Source](https://github.com/freeCodeCamp/how-to-contribute-to-open-source) repo (big fan of FreeCodeCamp ❤️ )

Really enjoyed the "Spotlight on Maintainers" presentation (shoutout to @abbycabs!) and looking forward to seeing other people's suggestions here as well 😄
